examples/experimental/i3s-17-and-debug/tile-debug.js

```diff
@@ -20,6 +20,11 @@ const REFINEMENT_TYPES = {
 
 const FLOAT_VALUES_FIXED_COUNT = 3;
 
+/**
+ * Return short tile info
+ * @param {object} tileHeader
+ * @returns {object} - short tile info for debugging purposes
```

Comment:
It would be useful to add an /** @typedef {{...}} TileDebugInfo */ type

Reply:
Yes. Will do in further PR.